### PR TITLE
Adding Chrysler, Proton, Saipa

### PR DESCRIPTION
**Pull request by bkissin**

### CODE REVIEW

File: brands/shop/car.json [hunk 1]
```diff
@@ -340,6 +359,18 @@
       "shop": "car"
     }
   },
+  "shop/car|سایپا": {
+    "countryCodes": ["ir"]
```
Comment (mds08011) [inline brands/shop/car.json]:
```suggestion
    "countryCodes": ["ir"],
```

**Comment by mds08011** [comment]
Thank you!

**Comment by Adamant36** [comment]
Are there actually "Chrysler" stores out there? I know there's places that sell sell Chryslers, but do they have their own self branded stores just called "Chrysler"?

**Comment by 1ec5** [comment]
It’s the same as with any other American car brand that has a dealership network. Chrysler is usually combined with other brands owned by the company (Jeep, Dodge, and Ram), but we have entries for those brands too.

[](https://commons.wikimedia.org/wiki/File:JEEP_Akita.jpg)

**Comment by Adamant36** [comment]
It's not really the name of the car dealership though (in this case I assume it would be "Something Something Chrysler Jeep Whatever"). I feel like in cases like this there should be a `carbrand=*` tag that's used instead and then stores are tagged with the local name. It shouldn't be assumed that all car dealerships with "Chrysler" signs out front are corporate though or shouldn't be tagged with the local dealerships name. Which you usually can't tell just with drive by sign tagging.

**Comment by bkissin** [comment]
I definitely get where @Adamant36 is coming from, which is why I was hesitant to add the other brands under the same umbrella corporation. With the exception of VW group brands (VW, SEAT, Skoda, Audi, etc.) many multi-brand automakers (partially because they often group brands together at dealerships) are not listed in the index. I don't know whether it would make sense to create individual entries for Dodge, Jeep, etc. unless there was an option to add multiple brands under the brand= section.

That being said, there is certainly a way to tag the brand, but keep the local name. Usually when I'm in iD, I'll update the tag, and then change the name from "Rimouski Honda" to "Honda" in order to activate the brand info, and then go back into the name section and re-add the local name information.

**Comment by bhousel** [comment]
per #2847 I'd prefer to avoid combining brands in the index for now.  

It's totally ok if mappers want to add several points and either leave them separate or merge them together (creating semicolon-delimited values),  or if mappers want to adjust the name tag to say "Whatever Honda".  iD supports both of these things.

**Comment by 1ec5** [comment]
Car brands were one of the original use cases for the `brand` tag. The dealerships I have in mind are the ones whose monument signs are emblazoned with the Chrysler, Jeep, and Dodge logos and nothing else. I’m not referring to, say, used car dealerships that happen to carry Chrysler cars but are unaffiliated with that company. Whether the dealership is corporate or franchised is a concern for `operator`, not `brand`.

It is the case that car dealerships usually have something else in the name, signage notwithstanding. We should encourage mappers to add this detail, but location-specific names are never by themselves a reason to omit entries from this index.

**Pull request merged**